packages/react-aria-components/docs/ColorSlider.mdx

```diff
@@ -0,0 +1,687 @@
+The value provided to either of these props should be a color string or <TypeLink links={docs.links} type={docs.exports.Color} /> object. The `channel` prop must also be provided to specify which color channel the slider should display. This must be one of the channels included in the color value, for example, for RGB colors, the "red", "green", and "blue" channels are available. For a full list of supported channels, see the [Props](#props) table below.
+
+### Controlled
+
+In the example below, the <TypeLink links={docs.links} type={docs.exports.parseColor} /> function is used to parse the initial color from an HSL string. This is passed to the `value` prop to make the `ColorSlider` controlled, and updated in the `onChange` event.
+
+```tsx example
+import {parseColor} from 'react-aria-components';
+
+function Example() {
+  let [value, setValue] = React.useState(parseColor('hsl(0, 100%, 50%)'));
+  return (
+    <>
+      <MyColorSlider
+        label="Hue (controlled)"
+        value={value}
+        onChange={setValue}
+        channel="hue" />
+      <p>Value: {value.toString('hex')}</p>
+    </>
+  );
+}
+```
+
+### HTML forms
+
+ColorSlider supports the `name` prop for integration with HTML forms. The value will be submitted as a number between the minimum and maximum value for the displayed channel.
+
+```tsx example
+<MyColorSlider
+  defaultValue="#7f0000"
+  channel="red"
+  name="red" />
+```
+
+## Events
+
+ColorSlider supports two events: `onChange` and `onChangeEnd`. `onChange` is triggered whenever the ColorSlider's handle is dragged, and `onChangeEnd`
+is triggered when the user stops dragging the handle. Both events receive a <TypeLink links={docs.links} type={docs.exports.Color} /> object
+as a parameter.
+
+The example below uses `onChange` and `onChangeEnd` to update two separate elements with the ColorSlider's value.
+
+```tsx example
+function Example() {
+  let [currentValue, setCurrentValue] = React.useState(parseColor('hsl(50, 100%, 50%)'));
+  let [finalValue, setFinalValue] = React.useState(currentValue);
+
+  return (
+    <div>
+      <MyColorSlider
+        value={currentValue}
+        channel="hue"
+        onChange={setCurrentValue}
+        onChangeEnd={setFinalValue} />
+      <p>Current value: {currentValue.toString('hsl')}</p>
+      <p>Final value: {finalValue.toString('hsl')}</p>
+    </div>
+  );
+}
+```
+
+## Creating a color picker
+
+### RGBA
+
+This example shows how you could build an RGBA color picker using four color sliders bound to the same
+color value in state. The <TypeLink links={docs.links} type={docs.exports.parseColor} />
+function is used to parse the initial color from a hex value, stored in state. The `value` and `onChange` props
+of ColorSlider are used to make the sliders controlled, so that they all update when the color is modified.
+
+```tsx example
+function Example() {
+  let [color, setColor] = React.useState(parseColor('#ff00ff'));
```

Comment:
this is hex but the title says rgba, not a big deal, though maybe we should wait to use the hex until we add support for the alpha channel? https://github.com/adobe/react-spectrum/pull/3302
instead we can use the `rgba(r, g, b, a)` syntax here

Reply:
yeah, but we're only sending in a color with three channels, I know it'll default to 1 for opacity, but maybe misleading name? or misleading value being input? all the other examples include it